distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -1,23 +1,199 @@
 #!/bin/sh
 
-SCRIPTDIR=$(dirname "$0")
+# CONFIGURATION OPTIONS
+# -----------------------------------------------------------------------------
```

Comment:
I wonder if "internally managed" configuration should be renamed to "unzip and go" or "unzip and run" configuration. Would that make more sense to users/administrators?